ios/FluentUI/Navigation/Shy Header/ShyHeaderController.swift

```diff
@@ -212,6 +222,8 @@ class ShyHeaderController: UIViewController {
 
     private func setupShyHeaderView() {
```

Comment:
nit: maybe we should just cache the nav item and reuse it `let navigationItem = contentViewController.navigationItem`